src/main/scala/io/smartcat/twicas/TrainJob.scala

```diff
@@ -26,14 +28,26 @@ object TrainJob extends App {
 
   val df = rdd.toDF
 
-  val pipelineProcesor = new PipelineProcessor()
+
   val textCleaner = new TextCleaner(List("text", "userDescription"))
   val tokenizer = FeatureTokenizer.make(List("text", "userDescription"))
   val hashingTF = FeatureHashTF.make(Map("text_t" -> 100, "userDescription_t" -> 100))
-  val result = pipelineProcesor.processAll(df, List(textCleaner, tokenizer, hashingTF))
+  val pipelineProcesorToTF = new PipelineProcessor(List(textCleaner, tokenizer, hashingTF))
+  val result = pipelineProcesorToTF.processAll(df)
 
   val idf = FeatureIDF.make(result, List("text_t_tf", "userDescription_t_tf"))
 
-  val preprocessPipeline = pipelineProcesor.processAll(df, List(textCleaner, tokenizer, hashingTF, idf))
+  val assembler = FeatureAssembler.make(List("text_t_tf_idf", "userDescription_t_tf_idf"))
+
+  val preprocessPipelinePreprocess = new PipelineProcessor(List(textCleaner, tokenizer, hashingTF, idf, assembler))
+  val res = preprocessPipelinePreprocess.processAll(df)
+
+  val modelSummaries = LogisticRegressionTweet.makeModels(res, res, List(0.5, 0.6), List(0.0), List(0.0))
```

Comment:
This is just showcase for tuning model. Will add possible values for thresholds and rest params later  when I make data set.